.github/workflows/autorelease.yml

```diff
@@ -9,6 +9,9 @@ on:
     tags:
       - 'v*'
           
+permissions:
+  contents: read
```

Comment:
Is that really sufficient? I think our release process also needs to write. But I'm not 100% sure.